cherrypy/_cpdispatch.py

```diff
@@ -46,10 +48,12 @@ def kwargs(self):
 
     @kwargs.setter
     def kwargs(self, kwargs):
+        """Setter for named kwargs."""
         cherrypy.serving.request.kwargs = kwargs
         return cherrypy.serving.request.kwargs
 
     def __call__(self):
+        """Call handler for PageHandler."""
```

Comment:
Would
```suggestion
        """Invoke an HTTP handler callable for PageHandler."""
```
be more accurate?

Reply:
Also, *I think* referencing the class using RST syntax should work:
```suggestion
        """Invoke an HTTP handler callable for :exc:`PageHandler`."""
```
(needs testing)